dependency.repos

```diff
@@ -13,6 +13,11 @@ repositories:
     type: git
     url: https://github.com/DroneBase/livox_ros2_driver.git
     version: 72cb73e
+  docker/humble-custom/livox_laser_simulation_ros2:
+    # not distributed
```

Comment:
What does this comment mean?

Reply:
It is not necessary. I copied lines from somewhere in this file (people, or velodyne, or velodyne_simulator), and forgot to remove the comment. 
I will remove the comment. Are these comment necessary for other packages?

Reply:
Oh, I remember that these (people, or velodyne, or velodyne_simulator) packages are not distributed by apt and added comment.
Livox is also not distributed, so please keep the comment.